src/Illuminate/Support/Collection.php

```diff
@@ -385,12 +385,16 @@ public function every($key, $operator = null, $value = null)
     /**
      * Get all items except for those with the specified keys.
      *
-     * @param  mixed  $keys
+     * @param  mixed|Collection  $keys
      * @return static
      */
     public function except($keys)
     {
-        $keys = is_array($keys) ? $keys : func_get_args();
+        if ($keys instanceof Collection) {
+            $keys = $keys->keys()->items;
```

Comment:
Isn't items protected?

Reply:
yes, it has direct access to it, but i would suppose the method call would be better?

Reply:
@lagbox *all()* method? probably yes